src/Microsoft.OData.Core/Json/ODataJsonWriterUtils.cs

```diff
@@ -248,5 +256,89 @@ private static void WriteInnerError(IJsonWriter jsonWriter, ODataInnerError inne
             // }
             jsonWriter.EndObjectScope();
         }
+
+        internal static void ODataValueToString(StringBuilder sb, ODataValue value)

```

Comment:
I think this static class is wrongly named because of the methods contained are only related to ODataError and writing the json for it. The other place is ODataInnerError itself which does not make sense to me. 


---
In reply to: [323952586](https://github.com/OData/odata.net/pull/1523#discussion_r323952586) [](ancestors = 323952586)